_posts/2023-06-30-july-10-downtime.md

```diff
@@ -16,11 +16,12 @@ Please be aware of the following outages:
 1. Back up
    1. SSO service, this means you will be unable to authenticate
    2. ESI production servers
+   3. ESI development servers will be cut off (kumo environment)
```

Comment:
maybe delete the "will be cut off (kumo environment) line, makes it a bit ambiguous us this is under the "back up" section.

Reply:
updated